### PR TITLE
Getters for numerator, denominator

### PR DESCRIPTION
**Pull request by micahkendall**

### CODE REVIEW

File: lib/aiken/math/rational.ak [hunk 1]
```diff
@@ -63,6 +63,56 @@ test new_1() {
...
 
+/// Get the numerator of a rational value.
+///
+/// ```aiken
+/// expect Some(x) = rational.new(2, 3)
+/// rational.numerator(x) == 2
+/// ```
+pub fn numerator(some: Rational) -> Int {
+  some.numerator
+}
```
Comment (KtorZ) [inline lib/aiken/math/rational.ak:74]:
```suggestion
pub fn numerator(self: Rational) -> Int {
  self.numerator
}
```

File: lib/aiken/math/rational.ak [hunk 2]
```diff
@@ -63,6 +63,56 @@ test new_1() {
...
+
+/// Get the denominator of a rational value.
+///
+/// ```aiken
+/// expect Some(x) = rational.new(2, 3)
+/// rational.denominator(x) == 3
+/// ```
+pub fn denominator(some: Rational) -> Int {
+  some.denominator
```
Comment (KtorZ) [inline lib/aiken/math/rational.ak:98]:
```suggestion
pub fn denominator(self: Rational) -> Int {
  self.denominator
```

**Review by KtorZ — APPROVED** [review]
Nice catch.

**Comment by fallen-icarus** [comment]
@micahkendall Not sure if it is appropriate for this PR but the documention for the `ceil` function is wrong; it is using `truncate` in the examples.

**Review by KtorZ — APPROVED** [review]

**Pull request merged**